mypy/myunit/__main__.py

```diff
@@ -6,4 +6,13 @@
 
 from mypy.myunit import main
 
+# In Python 3.3, mypy.__path__ contains a relative path to the mypy module
+# (whereas in later Python versions it contains an absolute path).  Because the
+# test runner changes directories, this breaks non-toplevel mypy imports.  We
+# fix that problem by fixing up the path to be absolute here.
```

Comment:
Interesting that this has not occurred before. I guess mypy so far has carefully avoided delayed imports like this. Hopefully once the fast parser is the only parser we can revert this hack (because I feel kind of bad about it).